BaseApiBundle/Repository/ApiClientRepository.php

```diff
@@ -20,4 +20,134 @@ public function findOneByKeyAndSecret($key, $secret)
     {
         return $this->findOneBy(array('key' => $key, 'secret' => $secret));
     }
+
+    /**
+     * @param array|null  $descriptionEntity
+     * @param array|null  $columns
+     * @param string|null $search
+     * @param array|null  $order
+     * @param int|null    $skip
+     * @param int|null    $limit
+     *
+     * @return array
+     */
+    public function findForPaginateAndSearch($descriptionEntity = null, $columns = null, $search = null, $order = null, $skip = null, $limit = null)
```

Comment:
This method name seems not enough explicit. Can you rename it to give it more sense ?